### PR TITLE
Move to backend-diagram-converter/core/src/test/resources

### PR DESCRIPTION
**Pull request by pierre-yves-monnet**
# Camunda Community Hub Pull Request Template

Structure to build a test case for conversion

## Description
The first use case is a simple "user task" process.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an existing open issue)
- [X] New feature (non-breaking change which adds functionality to an extension)
- [ ] Breaking change (fix or feature that would cause existing functionality of an extension to change)
- [ ] Documentation update (changes made to an existing piece of documentation)

## Checklist:
- [ ] My code adheres to the syntax used by this extension.
- [ ] My pull request requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **Camunda Community Hub** documentation.
- [ ] I have read the **Pull Request Process** documentation.
- [ ] I have added or suggested tests to cover my changes suggested in this pull request.
- [ ] All new and existing CI/CD tests passed.
- [ ] I will /assign myself this issue, and add the relevant [issue labels] to it if they are not automatically generated by Probot.
- [ ] I will tag @camunda-community-hub/devrel in a new comment on this issue if 30 days have passed since my pull request was opened and I have not received a response from the extension's maintainer.

### CODE REVIEW

**Comment by ingorichtsmeier** [comment]
Hi @pierre-yves-monnet, I thought about how to make the diagrams and the descriptions more prominent in this repository.

My assumptions: 
* collecting example processes with descriptions and may lead to redundant processes
* collecting example processes with description is important to explain other users how to migrate with maybe additional manual steps (eg. fill in the correlation key for message events).

What about using an extra project with a pom on the top level of this repository, named example-processes-to-migrate (or something better)?

Use the same structure as you had in docs.

Add a single test class for each example to test the migration (in the src/main/test/ folder). Why not access the diagram converter as a dependency here (in test scope)? 

These tests could only be executed manually. We don't need them to build a new version.

TL;DR: separate short and crisp examples to test the converter from more verbose, closer to real world examples with a detailed explanation. Find a structure that supports automated testing of the real world examples as well.

Would you like to try it out?

**Comment by pierre-yves-monnet** [comment]
According our discussion this morning, I created a new modules and move the test under this module.

**Review by ingorichtsmeier — APPROVED** [review]
All Good.
